pkg/util/validation/limits.go

```diff
@@ -625,6 +625,11 @@ func (o *Overrides) AlertmanagerMaxAlertsSizeBytes(userID string) int {
 	return o.getOverridesForUser(userID).AlertmanagerMaxAlertsSizeBytes
 }
 
+// GetOverridesForUser returns the per-tenant limits with overrides.
+func (o *Overrides) GetOverridesForUser(userID string) *Limits {
+	return o.getOverridesForUser(userID)
+}
+
```

Comment:
Maybe just export the function `getOverridesForUser` instead of create a new one just to export?